FFXIVClientStructs/FFXIV/Client/Graphics/Physics/BonePhysicsModule.cs

```diff
@@ -18,6 +18,7 @@ public struct BoneSimulators {
 public unsafe struct BonePhysicsModule {
     [FieldOffset(0x00)] public void* vtbl;
     [FieldOffset(0x10)] public Matrix4x4 SkeletonWorldMatrix;
+    [FieldOffset(0x40)] public Vector3 CharacterPosition;
```

Comment:
This is part of the `Matrix4x4` if you want to have just the vector3 component out, make a getter function for this instead.

Reply:
Oops yes, I miscalculated some of the sizes. I reverted this bit